scene/animation/animation_player.cpp

```diff
@@ -151,6 +151,7 @@ void AnimationPlayer::_notification(int p_what) {
 			if (!Engine::get_singleton()->is_editor_hint() && animation_set.has(autoplay)) {
 				set_active(true);
 				play(autoplay);
+				_process_animation(0);
```

Comment:
```suggestion
				seek(0, true);
```
To consist with other line like `_stop_internal()`.

Reply:
Thanks for the suggestion. See ed88521.